### ISSUE TITLE
[OpenMP] Enable libomptarget debugging for apt.llvm.org packages

### ISSUE THREAD

**Issue by ouankou** [llvm/llvm-project#134561]
I happened to notice that official packages of LLVM 20 have supported OpenMP GPU offloading directly if corresponding libraries (CUDA, ROCM, etc.) are installed. So there's no need to spend quite some time building from the source. However, it seems debugging is not enabled. Is it possible to enable it while packaging, please? Or is there some particular reason to keep it off? Thanks!

```bash
$ which clang
/usr/lib/llvm-20/bin/clang
$ clang -fopenmp -fopenmp-targets=nvptx64 test.c
$ LIBOMPTARGET_DEBUG=1 ./a.out
sum = 15050
```

**Comment by jhuber6** [comment]
We have no control over how distributions decide to build LLVM. I'd recommend you build it yourself using the CMake cache file at https://github.com/llvm/llvm-project/blob/main/offload/cmake/caches/Offload.cmake with it enabled. Something like
```console
> git clone <llvm-repo>
> cd llvm-project; mkdir build && cd build
> cmake ../llvm -C ../offload/cmake/caches/Offload.cmake -DLIBOMPTARGET_ENABLE_DEBUG=ON -G Ninja
> ninja install
```

**Comment by ouankou** [comment]
Thanks for the prompt response. I thought issues related to the official LLVM repo were managed here.
According to apt.llvm.org:
```
Bugs should be reported on the [LLVM bug tracker](https://github.com/llvm/llvm-project/labels/packaging) (label: packaging).
```
Of course, this is more like a feature request than a bug.
Thanks again for your information with details! I'll keep building the source for now.

**Comment by jhuber6** [comment]
I think that's intended more for bug reports so we can investigate if there's something wrong given the build used by the packagers. But I'm not an authority.
